src/error/go.rs

```diff
@@ -65,3 +66,27 @@ impl fmt::Display for GoResult {
         }
     }
 }
+
+impl GoResult {
+    /// this is like try_from but uses a fallback for GoResult::Other.
```

Comment:
```suggestion
    /// This is a wrapper around `impl TryFrom<GoResult> for Result<(), FfiError>` that uses a fallback
    /// when the conversion fails (i.e. in case of GoResult::Other).
```